buttons/colour.go

```diff
@@ -17,9 +17,8 @@ type ColourButton struct {
 }
 
 // GetImageForButton is the interface implemention to get the button's image as an image.Image
-func (btn *ColourButton) GetImageForButton() image.Image {
-	ButtonSize := 96
-	img := image.NewRGBA(image.Rect(0, 0, ButtonSize, ButtonSize))
+func (btn *ColourButton) GetImageForButton(btnSize int) image.Image {
+	img := image.NewRGBA(image.Rect(0, 0, btnSize, btnSize))
```

Comment:
Replace the `ButtonSize` variable with a `btnSize` parameter that gets passed in when the function is called. Might be a good idea to just make a `size` a field withing the `ColourButton` type